flow/scene_update_context.cc

```diff
@@ -222,6 +222,9 @@ SceneUpdateContext::ExecutePaintTasks(CompositorContext::ScopedFrame& frame) {
     surfaces_to_submit.emplace_back(std::move(task.surface));
   }
   paint_tasks_.clear();
+  alpha_ = 1.f;
+  topmost_global_scenic_elevation_ = 10.f;
```

Comment:
I curious where this `10.f ` comes from, and whether it's configurable.

Reply:
will pull out to constant

Reply:
Done